apps/tx-builder/src/hooks/useModal/useModal.tsx

```diff
@@ -0,0 +1,28 @@
+import { useCallback, useState } from 'react';
+
+function useModal(initialValue = false) {
```

Comment:
I created this custom hook to handle open/close Modals.

Let me know if its not useful!  :3